packages/components/data-table/src/header-cell.js

```diff
@@ -120,9 +120,13 @@ const HeaderCell = (props) => {
             <AngleUpDownIcon
               size="medium"
               color="surface"
-              id="nonActiveSortingIcon"
+              data-icon-state="inactive"
```

Comment:
Using "data attributes" for styles selectors instead of IDs, to avoid having duplicate IDs in the HTML document.